ludwig/modules/embedding_modules.py

```diff
@@ -409,9 +414,8 @@ def __init__(
         )
 
         if embedding_regularizer:
-            embedding_regularizer_obj = tf.keras.regularizers.get(
-                embedding_regularizer)
-            self.add_loss(lambda: embedding_regularizer_obj(self.embeddings))
+            self.add_loss(lambda: reg_loss(
+                self.embeddings, embedding_regularizer))
```

Comment:
Could you also add a test for `EmbedSequence` under `tests/ludwig/modules/test_embedding_modules.py`?

Reply:
Done.